Spreadsheet/index.html

```diff
@@ -33,223 +29,220 @@
                 <div id="table-input">
                     <input id="fxInput" type="text" value="">
                 </div>
-
             </div>
 
             <div class="table">
-                <div id="numbers">
-                    <div class="box" id="0"></div>
-                    <div class="box" id="1">1</div>
-                    <div class="box" id="2">2</div>
-                    <div class="box" id="3">3</div>
-                    <div class="box" id="4">4</div>
-                    <div class="box" id="5">5</div>
-                    <div class="box" id="6">6</div>
-                    <div class="box" id="7">7</div>
-                    <div class="box" id="8">8</div>
-                    <div class="box" id="9">9</div>
-                    <div class="box" id="10">10</div>
+                <div id="rowHeaders">
+                    <div class="cell" data-row="0"></div>
+                    <div class="cell" data-row="1">1</div>
+                    <div class="cell" data-row="2">2</div>
+                    <div class="cell" data-row="3">3</div>
+                    <div class="cell" data-row="4">4</div>
+                    <div class="cell" data-row="5">5</div>
+                    <div class="cell" data-row="6">6</div>
+                    <div class="cell" data-row="7">7</div>
+                    <div class="cell" data-row="8">8</div>
+                    <div class="cell" data-row="9">9</div>
+                    <div class="cell" data-row="10">10</div>
                 </div>
 
                 <div id="A">
-                    <div class="box" id="0">A</div>
-                    <div class="box" id="1"></div>
-                    <div class="box" id="2"></div>
-                    <div class="box" id="3"></div>
-                    <div class="box" id="4"></div>
-                    <div class="box" id="5"></div>
-                    <div class="box" id="6"></div>
-                    <div class="box" id="7"></div>
-                    <div class="box" id="8"></div>
-                    <div class="box" id="9"></div>
-                    <div class="box" id="10"></div>
```

Comment:
`id`s have to be unique. Here they are used multiple times across the code. Prefer `class` instead when an `id` is not unique. Here in this case the `id` is not used for styling but for indexing, in which case it's best to use a data attribute (e.g. `data-row`)